WS.js

```diff
@@ -22,7 +25,7 @@ function WS(uri,protocols){
         }
 
         newWS._WS_KEY=uri+protocols;
-
+        newWS.addEventListener = newWS.addListener;
```

Comment:
This seems like it will overwrite the browser functionality when run in the browser, did you test this?

If it does, just move it into the if(isNode) statement